source/dub/commandline.d

```diff
@@ -1009,6 +1010,9 @@ class TestCommand : PackageBuildCommand {
 		args.getopt("f|force", &m_force, [
 			"Forces a recompilation even if the target is up to date"
 		]);
+		args.getopt("r|recursive", &m_recursive, [
+			"Runs test on all packages including subpackages."
```

Comment:
Unfortunately, this command-line help seems different from fact.
This option recursive tests for dependencies, not subpackages.
(I'd rather test the subpackages recursive.)